src/duration-picker/DurationCard.js

```diff
@@ -101,9 +101,9 @@ export default class DurationCard extends Component {
         if (!currentUnitBlock) return null;
 
         return (
-            <div className="duration-picker">
+            <div className="param-row">
```

Comment:
Yes, but use mixins very sparingly. In that case likely not needed.
As an example what a mess sass might become is girddle that we use as grid in binary-static